### PR TITLE
Allow to disable hostname

### PR DESCRIPTION
**Pull request by johanneswuerbach**
This allows to disable sending a host tag by setting the host to `false`.

This helps in cases where statsd is not running on the same instance the tracked application is running.

### CODE REVIEW

**Comment by bryanmikaelian** [comment]
Thanks for the PR! @jderrett can I get your +1 on this or thoughts? I'm not opposed to merging this.

File: lib/librato.js [hunk 1]
```diff
@@ -630,13 +630,15 @@ exports.init = function librato_init(startup_time, config, events, logger)
       tags = config.librato.tags;
     }
     
-    // Set global measurement tags if they are defined.
-    if (config.librato.writeToLegacy && Object.keys(config.librato.writeToLegacy).length) {
+    // Write legacy metrics if enabled.
+    if (config.librato.writeToLegacy) {
```
Comment (mheffner) [inline lib/librato.js]: This seems unrelated?

Comment (mheffner) [inline lib/librato.js]: @bryanmikaelian did we have a broken check here?

Comment (johanneswuerbach) [inline lib/librato.js]: Sure, but the check seems wrong for a boolean value `Object.keys(true).length === 0`

Comment (johanneswuerbach) [inline lib/librato.js]: Removed.
